pom.xml

```diff
@@ -71,34 +64,16 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>email-ext</artifactId>
-            <version>2.69</version>
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>m2-extra-steps</artifactId>
```

Comment:
suspended for 8 years and automatically uninstalls itself:
https://github.com/jenkinsci/m2-extra-steps-plugin/commit/b84538a7cbc59639173aeeefcbd518c25d1d284b